index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <h1>Hello World, we are software developers!</h1>
+    <h1 id="title">Title</h1>
+    <p id="description">This is a survey form</p>
+    <form id="survey-form">
+      <label id="name-label" for="name">Name</label>
+      <input type="text" id="name" placeholder="Name" required />
+      <label id="email-label" for="email">Email</label>
+      <input type="email" id="email" placeholder="Email" required />
+      <label id="number-label" for="number">Number</label>
+      <input type="number" id="number" min="1" max="10" placeholder="Number" />
+      <select id="dropdown">
+        <option value="option 1">Option 1</option>
+        <option value="option 2">Option 2</option>
```

Comment:
these `option` tags should be more self descriptive.

Reply:
> * [ ]  these `option` tags should be more self descriptive.
- [ ] As I said in the previous review, kindly phrase your comments as `suggestions` instead of `orders`
- [ ] Always ensure that comments are clear and easy to understand. You could give the student some suggestions you think will help them understand better
For Example:

* This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
* This is a vague comment: *The README is not descriptive, why don't you make it better?*